module2/homework/insensitivePalindrom/palindrom.cpp

```diff
@@ -0,0 +1,18 @@
+#include "palindrom.hpp"
+
+#include <algorithm>
+#include <cctype>
+
+bool is_palindrome(std::string inputStr) {
+    inputStr.erase(std::remove_if(inputStr.begin(),
+                                  inputStr.end(),
+                                  [](auto strElement) { return !std::isalpha(strElement); }),
+                   inputStr.end());
+
+    std::transform(inputStr.begin(), inputStr.end(), inputStr.begin(),
+                   [](auto strElement) { return std::tolower(strElement); });
```

Comment:
this transform can be done without lambda function, it will be shorter then, but with lambda is also ok :)